CONTRIBUTING.md

```diff
@@ -108,11 +108,11 @@ to your valuable work:
 
 ### Running Tests
 
-- Running the `npm run test:node --ws` command from the root of the project will run all tests using node.
+- Running the `pnpm --recursive test:node` command from the root of the project will run all tests using node.
   - This is run via CI whenever a pull request is opened, or a commit is pushed to a branch that has an open PR
-- Running the `npm run test:browser --ws` command from the root of the project will run the tests in a browser environment
+- Running the `pnpm --recursive test:browser` command from the root of the project will run the tests in a browser environment
   - Please make sure there are no failing tests before switching your PR to ready for review! We hope to have this automated via a github action very soon.
-- You can also run `npm run test:node -w=packages/DIR` or `npm run test:browser -w=packages/DIR` from the root of the project to run tests for a single package. For example, to run the tests only for the `web5` package run `npm run test:node -w=packages/web5`.
+- You can also run `pnpm --filter=PACKAGE test:node` or `pnpm --filter=PACKAGE test:browser` from the root of the project to run tests for a single package. For example, to run the tests only for the `web5` package run `pnpm --filter=web5 test:node`.
```

Comment:
This typo predated this PR -- but might as well fix it.  There isn't a `web5` package in the monorepo so replaced with a package that exists.

```suggestion
- You can also run `pnpm --filter=PACKAGE test:node` or `pnpm --filter=PACKAGE test:browser` from the root of the project to run tests for a single package. For example, to run the tests only for the `dids` package run `pnpm --filter=dids test:node`.
```